### PR TITLE
Add recipe for SSAKE 4.0

### PR DESCRIPTION
**Pull request by npavlovikj**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ssake/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  sha256: {{ sha256 }}
+  url: https://github.com/warrenlr/{{ name }}/archive/v{{ version }}.tar.gz
+
+requirements:
+  run:
+    - perl-statistics-descriptive
```
Comment (bgruening) [inline recipes/ssake/meta.yaml]: Do you also need perl here?

File: recipes/ssake/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+sed -i.bak "s:/usr/bin/perl:/usr/bin/env perl:" ssake/SSAKE
+sed -i.bak "s:/usr/bin/perl:/usr/bin/env perl:" ssake/tools/*.pl
```
Comment (bgruening) [inline recipes/ssake/build.sh]: Can you try to patch this upstream as well?

Comment (npavlovikj) [reply recipes/ssake/build.sh]: Done :)

File: recipes/ssake/meta.yaml (hunk 1 above)
Comment (npavlovikj) [reply recipes/ssake/meta.yaml]: "perl" is installed as part of "perl-statistics-descriptive". Isn't this enough? If I specify "perl" is an independent requirement, there is a version incompatibility with the other packages, and the build fails.

**Pull request merged**